### ISSUE TITLE
GitVersion.SemVer: command not found

### ISSUE THREAD

**Issue by djbreen7** [GitTools/actions#61]
The code sample below produces the error `GitVersion.SemVer: command not found`

```yaml
  - task: UseGitVersion@5
    inputs:
      versionSpec: '5.x'

  ...

  - script: npm version $(GitVersion.SemVer) --no-git-tag-version
    displayName: 'NPM Version'
```

**Workaround**
Based on other issues, rolling back to `5.1.2` fixes the issue for me.

**Comment by arturcic** [comment]
Closing as it was fixed in 5.2.2

**Comment by djbreen7** [comment]
Thanks @arturcic . Sorry if this was the wrong place for the bug report. Looks like the support link on the visual studio marketplace has since been updated.
